torch_xla/csrc/init_python_bindings.cpp

```diff
@@ -345,6 +345,16 @@ void StepMarker(const std::string& device_str,
   }
 }
 
+std::string SyncLiveTensorGetHlo(const std::string& device_str,
```

Comment:
maybe `GetLiveTensorsStableHLO` and move it near to https://github.com/pytorch/xla/blob/master/torch_xla/csrc/init_python_bindings.cpp#LL364C13-L364C33